sphinxcontrib/spelling/filters.py

```diff
@@ -45,10 +45,78 @@ class ContractionFilter(Filter):
     """Strip common contractions from words.
     """
     splits = {
-        "won't": ['will', 'not'],
-        "isn't": ['is', 'not'],
-        "can't": ['can', 'not'],
-        "i'm": ['I', 'am'],
+        "i'm": ["I", "am"],
```

Comment:
The list is long enough now that it would be good to alphabetize it, just so it's easier to scan when adding something new.